Ezhil/README.md

```diff
@@ -0,0 +1,99 @@
+Ezhil
+=====
+
+Introduction
+============
+
+```
+எழில்: தமிழ் நிரலாக்க மொழி; முதன்முறை கணிப்பொறி நிரல் எழுதுகிற 
+தமிழ் மாணவர்களுக்கு இது உதவும்.
+
+எழில் மொழி  குழந்தைகளும்  , பள்ளி மாணவர்களும்  கட்ட்ருகொள்ளும் வகையில் உருவாக்கப்பட்ட ஒரு கணினி நிரல் மொழி.
+எழில் ஒரு செயல்முறை நோக்கு - (Imperative ) கணினி மொழி. இதை ரூபி , பைதான் போன்ற மொழிகளின் நடையில் அமைக்கபட்டது.
+எழில் மொழியில் நிபந்தனை கட்டளைகள், மடக்கு கட்டளைகள் என்றும்  எழுதலாம்; இதில் செயல்பாடுகள், செயல்குருகள் என்று கணினி நிரல்களை 
+பிரிக்கலாம். எழில்  மென்பொருள் அத்யாயம்  எண்  v0.8. பைதான் மொழி அடிப்படையில் உருவாக்கப்பட்டது.
+
+Ezhil-Lang : (Ezhil, is a fun Tamil programming language for K-12) Ezhil is 
+a procedural language with dynamic types, like Ruby/Python. Ezhil has a 
+pascal-like syntax, with for-end, while-end, if-elseif-else-end statements,
+break, continue and def-end for defining functions. Ezhil language is 
+implemented in a handwritten scanner and parser using the Python programming 
+language. Latest version of Ezhil-Language is v0.8.
+```
```

Comment:
You can put the back ticks immediately after the code and before the description